Modules/mmapmodule.c

```diff
@@ -397,6 +398,13 @@ is_resizeable(mmap_object *self)
             "mmap can't resize with extant buffers exported.");
         return 0;
     }
+#ifdef UNIX
+    if (!self->trackfd) {
+        PyErr_SetString(PyExc_TypeError,
```

Comment:
Shouldn't it be ValueError?

Reply:
It's `TypeError` for consistency with the `(self->access == ACCESS_WRITE) || (self->access == ACCESS_DEFAULT)` case.

Reply:
That makes the docs cumbersome, but it makes sense.